### PR TITLE
Remove feature toggle for SSCTs, VAULT_DISABLE_SERVER_SIDE_CONSISTENT_TOKENS

### PR DESCRIPTION
**Pull request by ncabatoff**

### CODE REVIEW

File: command/server.go [hunk 1]
```diff
@@ -1131,15 +1131,6 @@ func (c *ServerCommand) Run(args []string) int {
 	if envLicense := os.Getenv(EnvVaultLicense); envLicense != "" {
 		config.License = envLicense
 	}
-	if disableSSC := os.Getenv(DisableSSCTokens); disableSSC != "" {
-		var err error
-		config.DisableSSCTokens, err = strconv.ParseBool(disableSSC)
```
Comment (hghaf099) [inline command/server.go:1136]: It seems that there is also a reference to the server config for DisableSSCTokens. Should that also be removed?

File: vault/logical_system.go [hunk 2]
```diff
@@ -4791,7 +4778,6 @@ func (core *Core) GetSealStatus(ctx context.Context) (*SealStatusResponse, error
 		ClusterID:    clusterID,
 		RecoverySeal: core.SealAccess().RecoveryKeySupported(),
 		StorageType:  core.StorageType(),
-		Warnings:     core.getStatusWarnings(),
```
Comment (hghaf099) [inline vault/logical_system.go:4794]: nit: This Warnings seems to be used only for SSCT. It make sense to me to keep it around, but I was wondering if the corresponding struct should also be modified?

**Review by hghaf099 — APPROVED** [review]
Looks good to me. A couple of nit questions.

File: vault/logical_system.go (hunk 2 above)
Comment (ncabatoff) [reply vault/logical_system.go:4794]: Modified how?

File: command/server.go (hunk 1 above)
Comment (ncabatoff) [reply command/server.go:1136]: From what I could see that server config isn't actually usable from the config file (`hcl:"-"`) so it shouldn't matter, right?  The jira advised not cleaning up the code at this time, so all I did was remove the ability for a user to set the toggle.

File: vault/logical_system.go (hunk 2 above)
Comment (hghaf099) [reply vault/logical_system.go:4794]: Like to remove the warnings all together.

File: command/server.go (hunk 1 above)
Comment (hghaf099) [reply command/server.go:1136]: True, but I am wondering why to keep that around if it is a dormant line of code. It makes sense though to keep the scope within what Jira is asking! Thanks for the context.

File: vault/logical_system.go (hunk 2 above)
Comment (ncabatoff) [reply vault/logical_system.go:4794]: Ok, but I'm not sure how to reconcile that with your saying "it makes sense to me to keep it around".  It made sense to me as well, we might want to add new warnings in future.

File: command/server.go (hunk 1 above)
Comment (ncabatoff) [reply command/server.go:1136]: Yeah, I was afraid if I started down that path I wouldn't stop :)

File: vault/logical_system.go (hunk 2 above)
Comment (hghaf099) [reply vault/logical_system.go:4794]: Sorry for the confusion. I just wanted to learn your thought process on keeping that field around which as you mentioned is for potential future use.

**Pull request merged**